contracts/ToucanRegenBridge.sol

```diff
@@ -21,7 +21,7 @@ contract ToucanRegenBridge is Ownable, Pausable {
 
     // @dev mapping TCO2s to burnt tokens; acts as a limiting
     // mechanism during the minting process
-    mapping(address => uint256) tco2Limits;
+    mapping(address => uint256) public tco2Limits;
 
     // @dev address of the bridge wallet authorized to issue TCO2 tokens.
     address public regenBridge;
```

Comment:
I've been pondering about the name for a bit now, and I think it's slightly confusing with the name of this contract. I would propose to rename it to `regenBridgeController`.

Reply:
Makes sense, I was also slightly annoyed by it, will change in a follow-up.

Reply:
Opened https://github.com/regen-network/toucan-bridge/pull/10